### ISSUE TITLE
Give students post rather than pre week reading

### ISSUE THREAD

**Issue by oliverjam** [foundersandcoders/master-reference#364]
I'm not sure if this has been considered before, but it might be valuable to provide reading for students to do the weekend _after_ they have studied a topic, rather than before.

The problems with pre-week reading seem to be:

1. Some people won't be able to do it and will be at a disadvantage
2. If it is simple enough to understand before the students have been taught anything it probably makes the intro workshops for that week boring for anyone who did the reading.
3. Often good resources cover too much/are too advanced for students who haven't even been introduced to a topic yet.

Post-week reading doesn't have any of these problems:

1. It's an additional thing to reinforce or supplement the week they just did, so there is no disadvantage if people don't have the time to do the reading.
2. It won't affect the workshops at all.
2. It can be a more in-depth look at a topic that we potentially couldn't fit into the week.

The mentors for each week could pick their favourite couple of articles and send them to the students at the end of the week (e.g. I would have loved to read [this](https://medium.com/javascript-scene/what-every-unit-test-needs-f6cd34d9836d) right after testing week).

**Comment by jsms90** [comment]
@oliverjam Good points! We should be encouraging curiosity and never stunting it, so providing extra material is great Hopefully, students will be curious and _want_ more. When they do, the community should be able to provide an archive of resources that prove over time to be of most value to students at this point in their learning.

As I understand it, many students don't seem to be aware of the `resources.md` file that exists in every week of the `master-reference`. The intention was for the most useful supporting material to go in there, so that we can share the best recommendations. That seems like a perfect place to add your [article on testing](https://medium.com/javascript-scene/what-every-unit-test-needs-f6cd34d9836d) :+1:

So the question is: when/how do we point people to it? As I see it, the main objection to pre-reading (or post-reading) is about FAC implying that students can and should work over the weekend during an already intense course. People will do extra reading if they want to, and if they do, they might be at a technical advantage compared to the others (unless they share their knowledge). But this implication _from FAC_ that people _should_ spend their free time on this is the problem. It can cause pressure and have a negative impact on the self-esteem of those who feel they need a break. In turn, this can divide a cohort, not just in ability, but crucially in confidence.

**Comment by oliverjam** [comment]
@jsms90 I think I agree with everything there. I was vaguely aware of `resources.md` for some weeks but completely forgot it was there most of the time.

Maybe we just need to be more consistent in nudging students towards it at the right times? For me at least being pointed there at the beginning of the week meant I never looked at it. I didn't want to read ahead right before intro workshops, and by the end of the week I'd forgotten it was there.

**Comment by jsms90** [comment]
I definitely agree with you here. If mentors are consistently mentioning the `resources.md` file at [the beginning of their week](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/week-schedule.md#general-week-schedule), then people are more likely to remember that they exist
> 10:00-10.15 - Introduction to gitbook, learning outcomes & project teams

The benefit of this approach is that some students find it helpful to accompany their learning with extra material at different times:
+ Some students prefer pre-reading because it helps to reduce anxiety and make you feel more prepared.
+ Doing a little reading in the afternoons, while you're in the week, can help give students an opportunity to put what they've learned into practice within their project. It can also give people something extra to demonstrate during presentations.
+ Reading around the week's topics during the week can also lead to more sharing of knowledge. If, for example, a student reads something cool about testing during testing week, and then brings it to the group, the group might be more receptive because their minds are already on that topic.
+ Post-reading, as you've said is a great way to keep building on your knowledge and cementing it after you've finished with the workshops. Continual growth :tada: 

Basically I'm just an advocate of pointing it out, having our community actively curate a really really solid resources list to accompany each week, and giving students the freedom of choice. I'm sure you can get the students excited about some of your resources during the first 15 mins of testing week @oliverjam :grinning:

**Comment by oliverjam** [comment]
@jsms90 I agree with everything again :) Let's make sure mentors go through the learning outcomes, schedule and resources at the beginning of their week.

**Comment by jsms90** [comment]
@oliverjam I'm re-opening the issue, because we should provide clarification around this for any/all future cohorts who ask!

Thanks for raising the issue :tada:
